playbooks/robusta_playbooks/krr.py

```diff
@@ -0,0 +1,195 @@
+import json
+import uuid
+from datetime import datetime
+from typing import Dict, List, Optional, Union, Literal
+
+from hikaru.model import Container, PodSpec
+from pydantic import BaseModel, ValidationError, validator
+
+from robusta.api import (
+    EnrichmentAnnotation,
+    ExecutionBaseEvent,
+    FileBlock,
+    Finding,
+    FindingSource,
+    FindingType,
+    MarkdownBlock,
+    ProcessParams,
+    RobustaJob,
+    ScanReportBlock,
+    ScanReportRow,
+    ScanType,
+    action,
+    to_kubernetes_name,
+)
+from robusta.core.model.env_vars import RELEASE_NAME
+
+
+class KRRObject(BaseModel):
+    cluster: Optional[str]
```

Comment:
are all the fields really manadatory?

Reply:
This is just exactly the format we get from the Job

Reply:
This is just exactly the format we get from the Job